e2e/ChannelApeClient.spec.ts

```diff
@@ -8,20 +8,26 @@ import { LogLevel } from 'channelape-logger';
 
 describe('ChannelApe Client', () => {
 
+  beforeEach((done) => {
+    setTimeout(() => done(), 100);
+  });
+
   describe('Given valid session ID', () => {
     const sessionId = getSessionId();
 
     const channelApeClient = new ChannelApeClient({
       sessionId,
-      logLevel: LogLevel.OFF
+      logLevel: LogLevel.OFF,
+      jitterDelayMsMaximum: 5000,
+      jitterDelayMsMinimum: 1000,
```

Comment:
Please adjust these to get the e2e tests to run a little faster:
      jitterDelayMsMaximum: 2000,
      jitterDelayMsMinimum: 500,